eth-connector/src/lib.rs

```diff
@@ -344,7 +344,9 @@ impl EthConnectorContract {
             if balance == 0 || force {
                 self.ft.accounts.remove(&account_id);
                 self.ft.total_supply -= balance;
-                Promise::new(account_id.clone()).transfer(self.storage_balance_bounds().min.0 + 1);
+                Promise::new(account_id.clone()).transfer(
+                    self.storage_balance_bounds().min.0.saturating_add(1),
```

Comment:
This is marked as `This is a nightly-only experimental API`
Why not just use the  `checked_add`?

Reply:
fixed: https://github.com/aurora-is-near/aurora-eth-connector/pull/76/commits/201548a2be6e99c6d021cca2b1f700806ede7db6

Reply:
@olga24912 Please update the PR naming